libraries/AP_HAL_Linux/Util.cpp

```diff
@@ -28,6 +28,8 @@ void Util::init(int argc, char * const *argv) {
     saved_argc = argc;
     saved_argv = argv;
 
+    _system_time_was_set = CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_NONE;
```

Comment:
Did you mean CONFIG_HAL_BOARD == HAL_BOARD_LINUX?

Reply:
ty, done.

Reply:
Why is this line necessary? We are saying system_time_is_set for linux_none, but that's not actually the case. If we initialize it to false, it won't change or hurt.

Reply:
Done.